compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -17391,6 +17391,10 @@ TR::Node * arraysetSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier
       {
       uint64_t fillVal = fill->getConst<uint64_t>();
       if ((fillVal & 0x0FFFFFFFFL) == ((fillVal >> 32) & 0x0FFFFFFFFL) &&
+#ifdef TR_TARGET_ARM64
+          /* 0 and -1 can be efficiently loaded and stored as 64-bit data on AArch64. */
+          (fillVal != 0) && (fillVal != -1) &&
+#endif
```

Comment:
Please add a comment at the `#endif` line.
Refer to the guideline at https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#comment-conditional-compilation-directives-if-ifdef-ifndef-else-elif-endif

Reply:
Added a comment at the `#endif` line.